test/images/network-tester/webserver.go

```diff
@@ -227,32 +228,46 @@ func contactOthers(state *State) {
 		log.Printf("Server version: %#v\n", v)
 	}
 
+
+	for start := time.Now(); time.Since(start) < waitTimeout; time.Sleep(10 * time.Second) {
+		eps := getWebserverEndpoints()
```

Comment:
Good point. But doing this in the test still does not fix the underlying assumption in the image though.

The main problem with the container is that it starts contacting its peers immediately after it comes up. It is hard to guarantee all containers starts around the same time.